src/Traits/LogsActivity.php

```diff
@@ -14,6 +13,8 @@
 {
     use DetectsChanges;
 
+    private $withoutActivityLogging = false;
```

Comment:
In my mind it's easier to reason about these kind of boolean, where they are named in a positive.

Rename this to `enableLoggingModelsEvents` (or something better) with a default value of `true`